cypress/e2e/fact_spec.cy.js

```diff
@@ -0,0 +1,27 @@
+describe('visiting the fact page', () => {
+
+  it('should show a random fact upon visit', () => {
+    cy.intercept('https://catfact.ninja/fact', {
+      status: 200,
+      fixture: 'cat-facts'
+    })
+    cy.visit('http://localhost:3000/')
+    cy.get('button[class=fact-btn]').click()
+    cy.get('p').contains('Cats can see like wicked good')
+  })
+
+  it('should have two buttons', () => {
+    cy.visit('http://localhost:3000/fact')
+    cy.get('button[class=fav-card-btn]')
+    cy.get('button[class=new-fact-btn]')
+  })
+
+  it('should display error if there it does not get fact and provide a path back home', () => {
```

Comment:
Wording is a bit unclear. May need refactoring.

Reply:
Good catch. I will make that change for sure